doc/rdoc/markup_reference.rb

```diff
@@ -584,83 +615,197 @@
 #
 #   For C code, the directive may appear in a stand-alone comment
 #
-# === Text Markup
+# ==== Text Markup
 #
-# Text in a paragraph, list item (any type), or heading
-# may have markup formatting.
+# Text markup is metatext that marks text as:
 #
-# ==== Italic
+# - Italic.
+# - Bold.
+# - Monofont.
 #
-# A single word may be italicized by prefixed and suffixed underscores.
+# Text markup may contain only one type of nested block:
 #
-# Examples:
+# - More text markup:
+#   italic, bold, monofont.
```

Comment:
Nit: I'm not a fan of lists with a single element. I think these should just be a single sentence.

Reply:
There are two of these;  I used a bullet to be parallel with multi-item lists of nested blocks.